app/services/feeds/import.rb

```diff
@@ -3,13 +3,13 @@
 # => add Feeds::ValidateFeedUrl to validate a single feed URL
 module Feeds
   class Import
-    def self.call
-      new.call
+    def self.call(users: nil)
+      new(users: users).call
     end
 
-    # TODO: add `users` param
-    def initialize
-      @users = User.where.not(feed_url: [nil, ""])
+    def initialize(users: nil)
+      # using nil here to avoid an unnecessary table count to check presence
+      @users = users.nil? ? User.with_feed : users
```

Comment:
Assuming `users` won't be a literal `false`, this could change to the following which feels slightly more idiomatic in Ruby. 

```suggestion
      @users = users || User.with_feed
```

Reply:
Ah ah the `.nil?` is a remnant of me using `users: Users.none` as the default but yeah, we can change that